bundles/org.eclipse.orion.client.debug/web/orion/debug/nls/root/messages.js

```diff
@@ -41,5 +41,6 @@ define({ //Default message bundle
 	"UncaughtExceptions": "Uncaught Exceptions",
 	"Add": "Add",
 	"Clear": "Clear",
-	"Maximize": "Toggle Maximize"
+	"Maximize": "Toggle Maximize",
+	"Close": "Close"
```

Comment:
I am not sure this is the right messages file if the Close is for the notification area

Reply:
It's for the Close in org.eclipse.orion.client.debug/web/orion/debug/debugDeploymentWizard.html